docs/hopr-documentation/docs/developers/tutorial-hello-world.md

```diff
@@ -0,0 +1,224 @@
+```
+Basic Xl5MT0NBTC10ZXN0aW5nLTEyM15e
+```
+
+If you sent a successful request, the response will look something like this:
+```
+{
+  "nativeAddress": "0x3a54dDE3ee5ACfd43C902cbecC8ED0CBA10Ff326",
+  "hoprAddress": "16Uiu2HAmE9b3TSHeF25uJS1Ecf2Js3TutnaSnipdV9otEpxbRN8Q"
+}
+```
+
+## Send messages
+
+The HOPR protocol allows you to send private messages across nodes, by using nodes as relayers. Each message “hops” (hence the
+name “HOPR”) until it reaches its final destination, its contents known only to the final recipient. This is done automatically
+by HOPR nodes, which pick a random path to send your message to your recipient.
+
+The path consists of at least `2` nodes that have channels openned between each others. For instance, to send a message to `node 2`
+from `node 1`, the path a message could take is `node 1 -> node 4 -> node 3 -> node 2`.
+
+We'll try doing this, we'll use the REST API from `node 1` and send a message to `node 2`, which we'll be able to see via our
+WebSocket client connection.
+
+### 1. Obtaining the address (`PeerId`) of `node 2`
+
+Using `node 2`, type in your terminal with `websocat` running or WebSocket client interface the following command:
+
+```
+address
+```
+
+You should see a response like the following:
+
+```
+{"type":"log","msg":"admin > address\n","ts":"2022-02-02T19:17:48.431Z"}
+{"type":"log","msg":"HOPR Address:  16Uiu2HAmKhrwGWcvaZ3ic5dgy7oFawmnELJGBrySSsNo4bzGBxHW\nETH Address:   0x4cD95E1deF16D5913255Fe0af208EdDe2e04d720","ts":"2022-02-02T19:17:48.435Z"}
+```
+
+As you can see, the address AKA PeerId of `node 2` is `16Uiu2HAmKhrwGWcvaZ3ic5dgy7oFawmnELJGBrySSsNo4bzGBxHW`. You can use that
+information to send a message from `node 1`. Make sure to keep your WebSocket client connected to see any messages to `node 2`
+
+### 2. Sending a message from `node 2` from `node 1`:
+
+To send a message from `node 1` to `node 2`, we need to use `node 1`'s REST API, particularly the `/messages` endpoint. Using `curl`
+or any other HTTP client, send the following request:
+
+```
+echo -n $apiToken | base64 | xargs -I {} curl -s -H "Authorization: Basic {}" \
+-H 'Content-Type: application/json' \
+-d '{"body":"Hello world","recipient":"16Uiu2HAmKhrwGWcvaZ3ic5dgy7oFawmnELJGBrySSsNo4bzGBxHW"}' \
+$HOPR_NODE_1_HTTP_URL/api/v2/messages
+```
+
+In the terminal of `node 2`, you will see something similar to this:
+
+```
+{"type":"log","msg":"#### NODE RECEIVED MESSAGE [2022-02-03T21:48:13.845Z] ####","ts":"2022-02-03T21:48:13.845Z"}
+{"type":"log","msg":"Message: Hello world","ts":"2022-02-03T21:48:13.846Z"}
+{"type":"log","msg":"Latency: 668ms","ts":"2022-02-03T21:48:13.846Z"}
+{"type":"message","msg":"Hello world","ts":"2022-02-03T21:48:13.847Z"}
+```
+
+Congratulations! You have sent your first message using the HOPR protocol!
+
+### 3. Extra: Channels and tickets
+
+You can not send an unlimited amount of messages. Each message requires `ticket`s, headers with signatures for claiming
```

Comment:
Applies to multi-hop messages, 0-hop can be sent infinitely.

Reply:
Correct, but the message here is related to the REST API. Valid point though, worth adding it to a footnote.

Reply:
Included in [8abc8fb](https://github.com/hoprnet/hoprnet/pull/3397/commits/8abc8fb6f94655355f5de7df84739c33cc3e5b8e)